cmake/modules/autopas_antlr4cpp.cmake

```diff
@@ -1,101 +1,113 @@
-message(STATUS "antlr4cpp - using bundled version")
-
-include(ExternalProject)
-
-# check if uuid-dev is installed on the system, since this is a dependency of antlr4cpp
-find_package(PkgConfig)
-pkg_check_modules(UUID QUIET uuid)
-
-# if uuid-dev was not found on system we install it locally
-if (NOT UUID_FOUND)
-    message(STATUS "UUID not found - using bundled version")
-
-    set(LIBUUID_INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/uuid/install")
-    set(LIBUUID_PKGCONFIG_DIR ${LIBUUID_INSTALL_DIR}/lib/pkgconfig:$ENV{PKG_CONFIG_PATH})
-    set(LIBUUID_LIBRARY_DIR ${LIBUUID_INSTALL_DIR}/lib:$ENV{LIBRARY_PATH})
-    set(UUID_CONFIG_PARAMS "--prefix=${LIBUUID_INSTALL_DIR}")
-    ExternalProject_Add(
-        uuid_bundled
-        PREFIX ${CMAKE_CURRENT_BINARY_DIR}/uuid
-        URL ${PROJECT_SOURCE_DIR}/libs/libuuid-1.0.3.zip
-        URL_HASH MD5=9fd1e87682d24d6ca22941e0af339c8a
-        BUILD_IN_SOURCE TRUE
-        INSTALL_DIR "install"
-        CONFIGURE_COMMAND "./configure" ${UUID_CONFIG_PARAMS}
-        BUILD_COMMAND ${MAKE_EXE}
-    )
+set(AUTOPAS_ENABLE_RULES_BASED_TUNING
+        ON
```

Comment:
Personally, I would prefer the default to be off.
My reasoning would be that rules based tuning does not really work out of the box anyway. The user has to generate or write a rule file first. Someone who goes through this effort can then also easily turn on this option.
On the other hand, most new users won't need this and when it is OFF by default they just don't have to worry about it.